pages/project-version/pdf/views/nts/v1.jsx

```diff
@@ -74,7 +74,11 @@ export default function SchemaV1() {
 
       <div className="q-and-a">
         <h3>Key words</h3>
-        <p>No answer provided.</p>
+        <p>{
+          (version.keywords || []).length
+            ? (version.keywords || []).join(', ')
```

Comment:
```suggestion
            ? version.keywords.join(', ')
```
same as above, no point checking twice?

Reply:
Valid point.